src/libs/Navigation/types.ts

```diff
@@ -95,7 +95,9 @@ type SettingsNavigatorParamList = {
     [SCREENS.WORKSPACE.REIMBURSE]: {
         policyID: string;
     };
-    [SCREENS.WORKSPACE.RATE_AND_UNIT]: undefined;
+    [SCREENS.WORKSPACE.RATE_AND_UNIT.ROOT]: undefined;
+    [SCREENS.WORKSPACE.RATE_AND_UNIT.RATE]: undefined;
+    [SCREENS.WORKSPACE.RATE_AND_UNIT.UNIT]: undefined;
```

Comment:
`policyID` should be added for both `.RATE` and `.UNIT` since it will be passed as a URL parameter.